native/cocos/renderer/gfx-validator/DescriptorSetValidator.cpp

```diff
@@ -116,7 +116,9 @@ void DescriptorSetValidator::updateReferenceStamp() {
 
 void DescriptorSetValidator::bindBuffer(uint32_t binding, Buffer *buffer, uint32_t index) {
     CC_ASSERT(isInited());
-    CC_ASSERT(buffer && static_cast<BufferValidator *>(buffer)->isInited());
+    auto *vBuffer = static_cast<BufferValidator *>(buffer);
+    CC_ASSERT(buffer && vBuffer->isInited());
+    CC_ASSERT(vBuffer->isValid() && "Buffer View Expired");
```

Comment:
"Buffer View Expired" is not needed, or just treat it as comment?

Reply:
Assert shows the condition if failed. Just a Hint.